text/069-coq-roadmap.md

```diff
@@ -0,0 +1,444 @@
+- Title: Coq roadmap
+
+- Drivers: Nicolas Tabareau (@tabareau), Théo Zimmermann (@Zimmi48)
+
+----
+
+# Summary
+
+This CEP aims to establish a short term roadmap for the Coq project (while
+also stating the need for a medium and long term roadmap as well).
+
+It highlights both:
+
+- what are considered as priorities for the short term future,
+- what priorities we commit to make progress on, based on the available
+  resources.
+
+Resources mean availability of persons to conduct the work. For technical axes,
+requiring changes in the Coq system, it includes availability of persons to
+review and merge the changes.
+
+# Motivation
+
+Producing a roadmap for the Coq project is important for several reasons:
+
+- It helps the community to know what to expect from the Coq project in the
+  future, in particular what changes can reasonably be expected to become
+  part of next releases, and what changes are more likely to be delayed
+  because of lack of resources.
+
+- It helps Coq developers to choose priorities together, and to work more
+  efficiently toward those by making sure that their work will be supported
+  by other developers, including reviewers.
+
+- It helps contributors to know in what areas their contribution is likely
+  to receive the most support from the Coq developers.
+
+- It helps highlighting other priorities where progress is not possible for
+  lack of resources, and where the Coq project should aim to find
+  more resources.
+
+# Detailed design
+
+The Coq roadmap is declined at three levels: short, medium and long term.
+Each of these levels address different needs, but they should be checked
+for consistency with each other.
+
+- The short term roadmap is focused on priorities on which we are committed
+  to progress in the coming months and that are thus the most likely to
+  actually happen. We define a commitment from the Coq team as a commitment
+  of several (at least two) contributors, including at least one relevant
+  maintainer to review the work proposed by other contributors. The work
+  can include implementation work when the design is already clear, work on
+  the design from ideas that are well understood, or even exploratory work,
+  but we should only list the parts that are likely to converge in the
+  specified time-frame (i.e., if it is not clear if exploratory work will
+  lead to a solution that can be implemented, we should only commit on the
+  exploratory work).
+
+- The medium term roadmap is focused on areas where we see important progress
+  being needed, which could take several years to complete, and for which we
+  do not always have the ressources needed. It is used to guide the renewal
```

Comment:
```suggestion
  do not always have the resources needed. It is used to guide the renewal
```

Maybe call this simply "unresourced improvements" instead of "medium term roadmap"?  I'm sure there are many unresourced tasks that won't require "several years to complete".